Assets/World/WorldThing.gd

```diff
@@ -42,8 +42,8 @@ func _ready() -> void:
 	set_rotation_step(rotation_step)
 	set_rotation_degree(rotation_degree)
 
-	if not Engine.is_editor_hint():
-		_recalculate_translation()
+	Global.camera_rotate_left.connect(Callable(self, "_on_camera_rotate_left"))
+	Global.camera_rotate_right.connect(Callable(self, "_on_camera_rotate_right"))
```

Comment:
Methods need to be guarded as they will (similar to `ZoomButton.gd` with active `@tool`) try to call singleton members which are unavailable in-editor.
```suggestion
    if not Engine.is_editor_hint():
	    Global.camera_rotate_left.connect(Callable(self, "_on_camera_rotate_left"))
	    Global.camera_rotate_right.connect(Callable(self, "_on_camera_rotate_right"))
```